{{cookiecutter.project_dirname}}/{{cookiecutter.django_settings_dirname}}/urls.py

```diff
@@ -18,11 +18,32 @@
 from django.contrib import admin
 from django.urls import include, path, re_path
 from django.views.static import serve
+from rest_framework.decorators import api_view, permission_classes
+from rest_framework.response import Response
 
 admin.site.site_header = admin.site.site_title = "{{ cookiecutter.project_name }}"
 
+
+@api_view()
+@permission_classes([])
+def get_healthcheck(request):
+    """Send a health check response."""
+    return Response({"status": "ok"})
```

Comment:
A me personalmente garberebbe di più una class-based view nel suo `views.py`, in questa stessa directory.